src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,57 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Representation of a deadline with a time.
+ */
+public class Deadline extends Task {
+
+    protected LocalDate by;
+
+    /**
+     * Deadline constructor.
+     *
+     * @param description desc
+     * @param a           a
+     */
```

Comment:
I think it would helpful to be more specific with the description of the parameters.